### ISSUE TITLE
 [Security issue need investigation] Firebase allows for easy misconfiguration of security rules with zero warnings

### ISSUE THREAD

**Issue by gpdevsoft** [FilenCloudDienste/filen-mobile#374]
TLDR:
- Firebase allows for easy misconfiguration of security rules with zero warnings

- This has resulted in hundreds of sites exposing a total of ~125 Million user records, including plaintext passwords & sensitive billing information
source [https://env.fail/posts/firewreck-1/](url)

[https://github.com/0xbigshaq/firepwn-tool](url) 
this is an open source tool that can be use by developers to checking if the Firestore DB can be accessed without authentication, here you can see an explanation and demonstration of how to use it 
[https://appsec-labs.com/portal/firebase-applications-the-untold-attack-surface/](url)

**Comment by Dwynr** [comment]
Not sure why you are posting this, but we do not use Firestore. Firebase libs are only included due to FCM for push notifications. Maybe check out the source first.
